### ISSUE TITLE
Input video eroor

### ISSUE THREAD

**Issue by junaidhocanemuzamal** [PJunhyuk/people-counting-classification#16]
File "/usr/local/lib/python3.6/site-packages/moviepy/video/io/ffmpeg_reader.py", line 283, in ffmpeg_parse_infos
    line = [l for l in lines if keyword in l][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

[mov,mp4,m4a,3gp,3g2,mj2 @ 0x2ea3700] Format mov,mp4,m4a,3gp,3g2,mj2 detected only with low score of 1, misdetection possible!
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x2ea3700] moov atom not found
testset/test_video_03f.mp4: Invalid data found when processing input

**Comment by gg-z** [comment]
I've encountered the same problem. But I find that the video I downloaded is broken. Maybe you should check whether the videos you downloaded is ok. 
Hope this could help you out.

**Comment by junaidhocanemuzamal** [comment]
Yes This problem occurred because the video files were not correct and movipy was not installed. The problem was solved after completing this. 
The network is still so slow it process a single frame in 10 minutes. so it useless for practical purpose.
